src/devSVG.cpp

```diff
@@ -80,79 +118,90 @@ inline std::string fontname(const char* family_, int face) {
   }
 }
 
-inline void write_escaped(FILE* f, const char* text) {
+inline void write_escaped(std::ostream* stream, const char* text) {
   for(const char* cur = text; *cur != '\0'; ++cur) {
     switch(*cur) {
-    case '&': fputs("&amp;", f); break;
-    case '<': fputs("&lt;",  f); break;
-    case '>': fputs("&gt;",  f); break;
-    default:  fputc(*cur,    f);
+    case '&': (*stream) << "&amp;"; break;
```

Comment:
It feels like there must be a better way to do this, but the write method doesn't take a std::string. @jjallaire, do you have any suggestions?